project/plugins.sbt

```diff
@@ -31,5 +31,23 @@ addSbtPlugin("com.eed3si9n"      % "sbt-assembly"      % "1.1.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header"        % "5.6.0")
 addSbtPlugin("com.github.sbt"    % "sbt-release"       % "1.1.0")
 addSbtPlugin("com.github.sbt"    % "sbt-pgp"           % "2.1.2")
-addSbtPlugin("com.github.sbt"    % "sbt-jacoco"        % "3.4.0")
 addSbtPlugin("com.github.sbt"    % "sbt-ci-release"    % "1.5.10")
+
```

Comment:
I know you didn't do it, but please remove one of the licence headers in this file. For some reason it is double.

Reply:
Fixed in commit c172f508a4148078977b1734bd3ce32d2e81a329.